src/test/java/jUnit/JsonParserTest.java

```diff
@@ -12,27 +11,35 @@
 import java.io.*;
 import java.nio.file.Files;
 import java.nio.file.Paths;
+import java.util.List;
+import java.util.stream.Collectors;
 import java.util.stream.Stream;
 
-import static jUnit.Utils.Constants.FILE_NAME_LIST;
 import static org.junit.jupiter.api.Assertions.*;
 
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
 public class JsonParserTest {
 
-    static Cart initialCart;
-    static JsonParser parser;
+    Cart initialCart;
+    JsonParser parser;
+
+    List<String> fileNameList = Stream.of(
+            "vadim-cart",
+            "vadim1-cart",
+            "vadim2-cart")
+            .collect(Collectors.toList());
 
     @BeforeAll
-    static void setUp() {
+    void setUp() {
         parser = new JsonParser();
-        initialCart = Utils.initCart(0);
     }
 
     @ParameterizedTest
-    @ValueSource(ints = {0})
+    @ValueSource(ints = {0, 1, 2})
```

Comment:
I would suggest the following:
 1) create test case that would simply verify write function by checking file with given name and extension is created in a certain folder
 2) make it non parametrized (as per comment above - no need to verify this function with almost same data set). Other than that, fileNameList will be replaced by single file name